full-service/src/json_rpc/wallet.rs

```diff
@@ -428,6 +432,17 @@ where
                 account_secrets: AccountSecrets::try_from(&account).map_err(format_error)?,
             }
         }
+        JsonCommandRequest::export_view_only_account_secrets { account_id } => {
+            let account = service
+                .get_view_only_account(&account_id)
+                .map_err(format_error)?;
+            JsonCommandResponse::export_view_only_account_secrets {
+                view_only_account_secrets:
+                    json_rpc::view_only_account::ViewOnlyAccountSecrets::try_from(&account)
```

Comment:
Optional/Equivalent: `(&account).try_into()?` or maybe even `account.try_into().map_err(format_error)?`

Reply:
Tried this and it works, but I'm curious why you prefer it. To me `try_from` seems to make more sense since it's explicitly stating that we're going to create this json_rpc `ViewOnlyAccountSecrets` from our `account` entity. Whereas `try_into` looks more ambiguous.